modules/rosa-sts-aws-requirements.adoc

```diff
@@ -0,0 +1,38 @@
+[id="rosa-sts-customer-requirements_{context}"]
+= Customer Requirements
```

Comment:
s/ Customer requirements / Customer requirements when using STS for deployment

or something like this - the heading needs to be explicit that this applies only to STS deployments.